src/ImageSharp/Formats/Png/Chunks/PngPhysical.cs

```diff
@@ -10,11 +10,11 @@ namespace SixLabors.ImageSharp.Formats.Png.Chunks;
 /// <summary>
 /// The pHYs chunk specifies the intended pixel size or aspect ratio for display of the image.
 /// </summary>
-internal readonly struct PhysicalChunkData
+internal readonly struct PngPhysical
```

Comment:
Not sure why you renamed this.

Reply:
I was trying to make all trunks type under the same naming style. Says PngHeader, PngTextData, **PngPhysical**, APngAnimationControl, APngFrameControl. Maybe I should rename them to Header, TextData, Physical, AnimationControl, FrameControl?